trust-the-process/first-step.html

```diff
@@ -1,5 +1,5 @@
 <head>
-  <link rel="stylesheet" type="text/css" href="../css/normalize.css">
+  <link rel="stylesheet" type="text/css" href="../css/normailze.css">
```

Comment:
This looks like a bug. `normalize` right?

Reply:
Yeah, I also double checked all documents with the CSS link.. the spelling was normalize.css and so when I changed it to normailze.css the documents I was working on started to come together alot smoother.

Reply:
I hope this did not affect anything at all if so please let me know.